### ISSUE TITLE
Syndi stealth item device locks need alternative locking method

### ISSUE THREAD

**Issue by ArtisticRoomba** [space-wizards/space-station-14#39528]
## Description
https://github.com/space-wizards/space-station-14/pull/39310 was merged - it offers a way to lock stealth items behind a passphrase so stuff won't work if you try to use it while locked. While cool, any mute character or heavy accent character can't take advantage of it.

Ideally it should be activatable by fingerprint or some other non-voice method. Note that it shouldn't prevent use if it detects the wrong fingerprint - this is something very close to firing pins and something we want to avoid.

On a technical side, the locking system uses the trigger system - so any implementation can make a new trigger and it will hook into the old system just fine.

**Comment by LaCumbiaDelCoronavirus** [comment]
What? Why was the PR merged in the first place if this issue was already brought up _before_ it was merged, but is now being brought up again?

**Comment by beck-thompson** [comment]
> What? Why was the PR merged in the first place if this issue was already brought up before it was merged, but is now being brought up again?

Your free to make your own methods, the PR was done in a way where its extremely easy too add new ones (Allowing for other locks was why the PR took so long to make). Ideally there would be a method to allow players to choose what lock they want to use (Maybe just a verb that allows you to choose and it then locks the item to that method?) and not just be forced to only use fingerprint / voice only.

As of right now, this is just an upgrade for syndicates and doesn't hurt them. Giving a method that works for 90% of players is better than waiting another month to get a perfect system that allows for multiple methods.

**Comment by beck-thompson** [comment]
> What? Why was the PR merged in the first place if this issue was already brought up before it was merged, but is now being brought up again?

Just made it even easier (#39532), now people just need to add more locks - simple ones like toggling the lock on examine should be easy to add.  See #39354 for ideas

I also just wanted to say that this comment was quite frustrating to me. I feel like no matter what I do people are not happy.

Don't add stuff quickly enough? People are frustrated that syndicates have no counterplay to sec "metagaming" now that the metashield is gone (see: https://forum.spacestation14.com/t/request-for-feedback-regarding-metashield-removal/22279/67) 

Add stuff quickly that it isn't fully finished? People get frustrated that it isn't finished and there are edge cases that aren't fully handled.

Both opinions are totally valid, but when half the playerbase is saying one thing and the other half is saying the opposite I feel like there is no possible way for me to win. Its quite demotivating and not fun! I have no clue what the solution is besides "Spend even more time reviewing and making PRs" which clearly isn't reasonable.

anyway, sorry for the rant, I'm not mad at you or anything I just had to get this off my chest

**Comment by FairlySadPanda** [comment]
Assigned to @beck-thompson due to [#39532](https://github.com/space-wizards/space-station-14/pull/39532)
